packages/api-reference/README.md

```diff
@@ -58,6 +58,7 @@ import '@scalar/api-reference/style.css'
       document.getElementById('api-reference').dataset.configuration =
         JSON.stringify(configuration)
     </script>
+
```

Comment:
Is this space intentional?

Reply:
Actually, yes. Not feeling strong about it, though.

Reply:
Up to you!